skimage/segmentation/_felzenszwalb.py

```diff
@@ -51,34 +50,11 @@ def felzenszwalb(image, scale=1, sigma=0.8, min_size=20):
     """
 
     if image.ndim == 2:
-        # assume single channel image
-        return _felzenszwalb_grey(image, scale=scale, sigma=sigma,
-                                  min_size=min_size)
+        image = image[...,None]
```

Comment:
True - as this originally called different code I made the minimal edits needed when removing the grayscale version... will update and commit